paywall/src/components/lock/ConfirmingFlag.js

```diff
@@ -83,4 +83,4 @@ export const mapStateToProps = ({ account, keys, transactions }, { lock }) => {
   }
 }
 
-export default withConfig(connect(mapStateToProps)(ConfirmingFlag))
+export default connect(mapStateToProps)(withConfig(ConfirmingFlag))
```

Comment:
Wow, does the order actually matter?

Reply:
I did not dig into why, but yes it mattered with the ConfirmingFlag component for some reason.